2015-11-29 16:45:56.7797,Error,request-error,[System.ArgumentException] "Nem található „/App.html” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:50:05.2953,Error,request-error,[System.ArgumentException] "Nem található „App” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:52:28.0442,Error,request-error,[System.ArgumentException] "Nem található „App.html” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:53:26.6291,Error,request-error,[System.ArgumentException] "Nem található „App.html” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:53:47.0128,Error,request-error,[System.ArgumentException] "Nem található „App” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:53:56.9078,Error,request-error,[System.ArgumentException] "Nem található „App” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
2015-11-29 16:57:13.0514,Error,request-error,[System.ArgumentException] "Nem található „/App” nevû útvonal az útvonalgyûjteményben.
Paraméter neve: name":    a következõ helyen: System.Web.Routing.RouteCollection.GetVirtualPath(RequestContext requestContext, String name, RouteValueDictionary values)
   a következõ helyen: System.Web.Http.WebHost.Routing.HostedHttpRouteCollection.GetVirtualPath(HttpRequestMessage request, String name, IDictionary`2 values)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.GetVirtualPath(HttpRequestMessage request, String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Route(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Routing.UrlHelper.Link(String routeName, IDictionary`2 routeValues)
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.Execute()
   a következõ helyen: System.Web.Http.Results.RedirectToRouteResult.ExecuteAsync(CancellationToken cancellationToken)
   a következõ helyen: System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- Verem vége nyomkövetés a kivétel elõzõ elõfordulási helyétõl kezdve ---
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   a következõ helyen: System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   a következõ helyen: System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
